packages/common/src/Adaptor.js

```diff
@@ -932,3 +932,49 @@ export function assert(expression, errorMessage) {
     return state;
   };
 }
+
+/**
+ * Outputs a message to the console.
+ * @public
+ * @function
+ * @example <caption>Log specific values</caption>
```

Comment:
Hey, sorry I'm sure I commented on this yesterday but it seems to have vanished?

Can we just have a single simple example for each of log and debug which is like `log($.data)`. We really don't need more than that. The existing example are slightly at odds with the result of the docs.